registry/storage/catalog.go

```diff
@@ -25,12 +25,12 @@ func (reg *registry) Repositories(ctx context.Context, repos []string, last stri
 		return 0, errors.New("no space in slice")
 	}
 
-	root, err := pathFor(repositoriesRootPathSpec{})
-	if err != nil {
-		return 0, err
+	root, errVal := pathFor(repositoriesRootPathSpec{})
```

Comment:
I think I would prefer just changing the named return to `err` than make the change here